finesse/gui/temp_control.py

```diff
@@ -255,7 +298,7 @@ def _create_controls(self) -> QGridLayout:
         self._power_bar.setTextVisible(False)
         self._power_bar.setOrientation(Qt.Orientation.Horizontal)
         layout.addWidget(self._power_bar, 1, 1, 1, 3)
-        layout.addWidget(QLineEdit("40"), 1, 4)
+        layout.addWidget(QLineEdit(), 1, 4)
```

Comment:
What does this `QLineEdit` do? Is it just a placeholder for now?

Reply:
This is the one I was referring to in our meeting yesterday. I _think_ it just shows the power level numerically, but I'm not certain as on the screenshot of the old GUI it shows 40 on both the hot and cold TC4820s, while the power level `QProgressBar`s are different (~40 and 0). It's updated in the TC4820 branch I'm yet to push - not sure why I bothered changing this here.

Reply:
I seeeee. I'd maybe just update it based on the power level then. We can always ask Jon about it when we show it to him.